migrations/migration_023.sql

```diff
@@ -134,5 +134,4 @@ UNION
 		organization__localized_texts.language = 'en'
 ;
 
-DROP INDEX IF EXISTS idx_fts_search_en;
```

Comment:
Dropping the materialized view (above) also drops the associated index, so this was just causing noise in the output with no effect.